reference/6/Microsoft.PowerShell.Utility/ConvertFrom-Csv.md

```diff
@@ -12,130 +12,148 @@ title: ConvertFrom-Csv
-# Delete header from $J
-PS C:\> $J = $J[0], $J[2..($J.count - 1)]
-PS C:\> $J | ConvertFrom-Csv -Header $Header
+
+```Output
+State         : Running
 MoreData      : True
 StatusMessage :
 Location      : localhost
-Command       : get-process
-State         : Running
+Command       : Get-Process
+StateInfo     : Running
 Finished      : System.Threading.ManualResetEvent
-InstanceId    : 6fcb6578-7f42-4d93-9f23-9937f6aac1a2
-SessionId     : 1
+InstanceId    : a259eb63-6824-4b97-a033-305108ae1c2e
+Id            : 1
 Name          : Job1
 ChildJobs     : System.Collections.Generic.List`1[System.Management.Automation.Job]
+BeginTime     : 12/20/2018 18:59:57
+EndTime       :
+JobType       : BackgroundJob
 Output        : System.Management.Automation.PSDataCollection`1[System.Management.Automation.PSObject]
 Error         : System.Management.Automation.PSDataCollection`1[System.Management.Automation.ErrorRecord]
 Progress      : System.Management.Automation.PSDataCollection`1[System.Management.Automation.ProgressRecord]
-Verbose       : System.Management.Automation.PSDataCollection`1[System.String]
-Debug         : System.Management.Automation.PSDataCollection`1[System.String]
-Warning       : System.Management.Automation.PSDataCollection`1[System.String]
-StateChanged  :
+Verbose       : System.Management.Automation.PSDataCollection`1[System.Management.Automation.VerboseRecord]
+Debug         : System.Management.Automation.PSDataCollection`1[System.Management.Automation.DebugRecord]
+Warning       : System.Management.Automation.PSDataCollection`1[System.Management.Automation.WarningRecord]
+Information   : System.Management.Automation.PSDataCollection`1[System.Management.Automation.InformationRecord]
 ```
 
-This example shows how to use the Header parameter of **ConvertFrom-Csv** to change the names of properties in the resulting imported object.
-
-The first command uses the **Start-Job** cmdlet to start a background job that runs a **Get-Process** command on the local computer.
-A pipeline operator (|) sends the resulting job object to the **ConvertTo-CSV** cmdlet, which converts the job object to CSV format.
-An assignment operator (=) saves the resulting CSV in the $J variable.
-
-The second command saves a header in the $header variable.
-Unlike the default header, this header uses MoreData instead of HasMoreData and State instead of JobStateInfo.
+The `Start-Job` cmdlet starts a background job that runs `Get-Process`. A job object is sent down
+the pipeline to `ConvertTo-Csv` and converted to a CSV string. The `$Header` variable contains a
+custom header that replaces the following default values: **HasMoreData**, **JobStateInfo**,
+**PSBeginTime**, **PSEndTime**, and **PSJobTypeName**. The `$J` variable contains the CSV string
+and is used to remove the default header. The `ConvertFrom-Csv` cmdlet converts the CSV string to a
+job object and uses the **Header** parameter to apply the `$Header` variable.
```

Comment:
`ConvertFrom-Csv` converts the CSV string into a PSCustomObject, not a Job object. The type is lost converting to CSV.

Reply:
Corrected the sentence to use PSCustomObject rather than job object.